src/content/docs/browser/single-page-app-monitoring/get-started/spa-v2.mdx

```diff
@@ -0,0 +1,106 @@
+* Aligned with SoftNav heuristics: The new agent adopts Google Chrome's soft navigation heuristics, providing more accurate interaction tracking and improved alignment with browser behavior.
+* Simplified interaction determination: Interactions are now defined as UI event (click, keydown, submit) -> route change -> DOM modification, offering a clearer and more efficient capture approach.
+Focus on key metrics: While reported data remains largely unchanged, the new agent no longer tracks JS execution or callback duration within interactions, streamlining the reported information.
+
+## Impact and user experience
+
+Changes to the SPA monitoring include:
+
+* Reduced interaction durations: Expect significantly shorter interaction durations, particularly for route changes. Initial page load interactions should remain stable.
+* Potential event disassociation: You might observe previously associated `AjaxRequest` and `JavascriptError` events becoming disassociated from interactions, reflecting the focus on UI-driven interactions.
+* API compatibility: The current SPA API remains entirely compatible, ensuring seamless integration with your existing code.
+* Enhanced control: The new [`newrelic.interaction({ waitForEnd: true })`](/docs/browser/new-relic-browser/browser-apis/interaction/) API function empowers you to customize interaction completion behavior if needed.
+* Deprecated `createTracer`: While still functional, the `createTracer` function is deprecated as it no longer keeps interactions open or times callbacks. Our data indicates minimal usage of this function.
+
+We're confident that Pro+SPAv2 represents a significant leap forward in SPA monitoring capabilities. By addressing past issues and embracing modern browser heuristics, we're committed to providing you with the best possible experience for monitoring your single-page applications.
+
+## Try out browser SPA monitoring 2.0 
+
+<Steps>
+
+    <Step>
+        ### Review the requirements
+
+        This feature is tested and supported inline with our standard browser support statement <link to it here>.
+
+
+    </Step>
+
+    <Step>
+        ### Add the feature flag to your browser agent
+
+        If your agent was installed with the APM method, contact our support team and we'll enable the feature flag on your account.
+        
+        If your agent was installed with NPM or the UI copy/paste method, you'll need to add a feature flag in the browser agent code:
+
+            1. Find the New Relic browser code in your webpage HTMl.
+
+            2. In the `init` configuration object, add the `soft_nav` feature flag. Here's an example:
+
+                ```js
+                <script type="text/javascript"> ;window.NREUM||(NREUM={});init={ …, feature_flags: ['soft_nav'] }:
+                ```
+
+            3. Deploy your app.
+
+        Need to disable this feature? Simply remove the feature flag.
+            
```

Comment:
We don't want to directly publish any way for them to manually enable or mess with their configurations atm. In fact, opt-in is restricted atm. I think this section has to change since this page for now will serve as info for the customers who do get it rather than also recruiting existing customers.